plugins/jira-dashboard/src/components/JiraTable/columns.tsx

```diff
@@ -53,6 +53,26 @@ export const columns: TableColumn[] = [
       );
     },
   },
+  {
+    title: 'Priority',
+    field: 'fields.priority.name',
```

Comment:
Looks great. Though I wonder if we can get the sorting to be after "importantness" instead of alphabetical order.

Reply:
Hi @GLundh ,

It is something I considered however even within our own org people have different options when it comes to Sorting whether they be Blocker, Major, Minor, Trivial or A,B,C,D

I am open to suggestions if you know of a nice way to handle all the cases that Jira can provide I am happy to implement.

Reply:
I'm not a Jira admin, so I'm not sure how everything looks behind the curtain, but isn't "Priority" a standard field with a predefined set of values? I was hoping we could achieve the same sorting that Jira-does:

![image](https://github.com/AxisCommunications/backstage-plugins/assets/300900/d32426a8-1fc4-49d2-8107-874341fbbd21)

These are sorted after severity, not alphabetically. It's just a suggestion, I'm fine with the PR if you don't want it.